doc/guides/writing-tests.md

```diff
@@ -231,9 +231,9 @@ assert.throws(
 For performance considerations, we only use a selected subset of ES.Next
 features in JavaScript code in the `lib` directory. However, when writing
 tests, for the ease of backporting, it is encouraged to use those ES.Next
-features that can be used directly without a flag in [all maintained branches]
-(https://github.com/nodejs/lts). [node.green](http://node.green/) lists
-available features in each release.
+features that can be used directly without a flag in
+[all maintained branches](https://github.com/nodejs/lts).
```

Comment:
I am not entirely sure, but considering that there are three links in the footer, I would prefer consistency. In most cases that makes sense, but barely for `[here](https://github.com/nodejs/testing/issues/27)`. I am fine if you leave it as it is, just wanted to suggest that, didn't know there were so many other "inline" links.